demo-ui/app/uk/gov/ons/addressIndex/demoui/controllers/ApplicationHome.scala

```diff
@@ -5,12 +5,18 @@ import play.Logger
 import play.api.mvc.{Controller, _}
 import play.api.mvc.Action
 import scala.concurrent.ExecutionContext
+import scala.language.implicitConversions
+import play.api.i18n.{I18nSupport, Messages, MessagesApi, Lang}
 
 @Singleton
-class ApplicationHome @Inject()(implicit ec : ExecutionContext) extends Controller {
+class ApplicationHome @Inject()(val messagesApi: MessagesApi)(implicit ec : ExecutionContext) extends Controller with I18nSupport {
 
-  def indexPage() : Action[AnyContent] = Action { implicit req =>
-    Logger.info("Rendering Index page")
-    Ok
+    def indexPage() : Action[AnyContent] = Action { implicit req =>
+      Logger.info("Rendering Index page")
+      req.getQueryString("lang") match{
```

Comment:
@RichardSmithONS Also please put the correct indentation in code, do not skip spaces. ` req.getQueryString("lang") match {`